src/Avalonia.Visuals/Media/GeometryDrawing.cs

```diff
@@ -68,7 +68,8 @@ public override void Draw(DrawingContext context)
 
         public override Rect GetBounds()
         {
-            return Geometry?.GetRenderBounds(s_boundsPen) ?? Rect.Empty;
+            IPen pen = Pen ?? s_boundsPen;
```

Comment:
s_boundsPen has comment "// Adding the Pen's stroke thickness here could yield wrong results due to transforms." from 5 years ago. Sometime ago it was broken, and we discussed it's safer to merge it in master and see if there won't be a problem for users. Without backporting to a stable for now.

Reply:
@maxkatz6 , for now it is unusable in 0.10.14 like in this screenshot :
![image](https://user-images.githubusercontent.com/4248852/171811801-878de6e8-e9b9-4cbb-881d-7c69c4dad29c.png)

I can add this change in the master but the file moved to Avalonia.Base, do I need to create a second PR ? I'm not familiar with this ^^

Reply:
Yeah, I think there is no better way with github, but create another PR.

Reply:
> Also makes sense to add some simple tests.

I've add some tests (in Avalonia.RenderTests) :)